docs/sbft-formal-model/messages.dfy

```diff
@@ -17,7 +17,7 @@ module Messages {
   import opened HostIdentifiers
   import Network
 
-  type SequenceID = nat
+  type SequenceID = k:nat | 0 < k witness 1
```

Comment:
Does `witness 1` indicate that k=1 is a witness that instantiates this predicate?

Reply:
Yes, exactly. This is a change that we made because in our system we start committing from Sequence ID No:1. Previously in the model the **SequenceID** was **nat**, starting from 0 to infinity. Both ways the invariants are preserved.